codalab_bundle/readme.md

```diff
@@ -1,9 +1,17 @@
-Training dataset 
+## Usage  
+
+Just run :
+`python create_codalab_bundle.py`
+
+
+## Dataset 
+
+Training dataset :
 https://zenodo.org/record/6673844/files/train.zip?download=1
 
-Validation dataset 
-https://zenodo.org/record/6673574/files/validation.zip?download=1
+Validation dataset :
+https://zenodo.org/record/6673574
 
 
-Sample submission format 
+Sample submission format :
 https://raw.githubusercontent.com/EPFL-ENAC/topo-vnav-ml-challenge/main/data_challenge_preparation/data_sample/est.csv

```

Comment:
line 12-16 - This should be in another commit, e.g "Doc: Updated resources links". Or rename the commit fully "Updated documentation"